common/config/azure-pipelines/jobs/docs-ci.yaml

```diff
@@ -43,8 +44,11 @@ jobs:
         displayName: Setup git config
 
       - template: ../templates/docs-build.yaml
+        parameters:
+          workingDir: $(Pipeline.Workspace)/itwinjs-core
+          outputDir: $(Agent.BuildDirectory)/tempDocsBuild/public_build
 
       - publish: $(Agent.BuildDirectory)/tempDocsBuild/
         artifact: DocsBuild
         displayName: Publish Pipeline Artifact - DocsBuild
-        condition: and(succeeded(), or(eq(variables['Build.SourceBranch'], 'refs/heads/master'), startsWith(variables['Build.SourceBranch'], 'refs/heads/release')))
+        condition: and(succeeded(), or(true, eq(variables['Build.SourceBranch'], 'refs/heads/master'), startsWith(variables['Build.SourceBranch'], 'refs/heads/release')))
```

Comment:
This change is temporary while I test a deployment with the new changes, will be reverted.